arch/arm/soc/st_stm32/stm32l4/Kconfig.defconfig.stm32l496xx

```diff
@@ -0,0 +1,20 @@
+# Kconfig - ST Microelectronics STM32L496xx MCU
+#
+# Copyright (c) 2016 Open-RnD Sp. z o.o.
+# Copyright (c) 2016 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L496XX
```

Comment:
Please keep Flash size information ("XG") in SOC_STM32xx configuration flag:  SOC_STM32L496XG
It's important so we could differentiate SoC flash variants (L496XE only for now) in dts/arm/st/mem.h file
I'll submit a PR to update STM32 family according to this. Sorry for not seeing this earlier.